.kokoro

```diff
@@ -497,7 +467,6 @@ fi
 
 case "$DEPENDENCY_SYSTEM" in
   "bazel")      run_bazel ;;
-  "clang-format")  lint_clang_format ;;
```

Comment:
I'm likely mistaken, but this appears to be changing how the `.kokoro` script works. I thought this PR is just a refactoring?

Reply:
I've updated the PR description to clarify.

Reply:
Sorry, I still don't understand when the new file is being invoked.  Could you please point-out in the PR where the new script is invoked?

Reply:
Updated the PR description with links to the kokoro job configurations.